src/List.js

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { mapToCssModules, tagPropType } from './utils';
+
+const propTypes = {
+  tag: tagPropType,
+  className: PropTypes.string,
+  cssModule: PropTypes.object,
+  type: PropTypes.string,
+  innerRef: PropTypes.oneOfType([PropTypes.object, PropTypes.string, PropTypes.func])
```

Comment:
Let's use `forwardRef` instead of `innerRef` since this is more suited for function components:
https://reactjs.org/docs/forwarding-refs.html

So below would be
`const List= React.forwardRef((props, ref) => ...`